tests/components/mhz19/test_sensor.py

```diff
@@ -8,130 +9,113 @@
     TEMP_CELSIUS,
     TEMP_FAHRENHEIT,
 )
-from homeassistant.setup import setup_component
+from homeassistant.setup import async_setup_component
 
 from tests.async_mock import DEFAULT, Mock, patch
-from tests.common import assert_setup_component, get_test_home_assistant
+from tests.common import assert_setup_component
 
 
-class TestMHZ19Sensor(unittest.TestCase):
-    """Test the MH-Z19 sensor."""
-
-    hass = None
+async def test_setup_missing_config(hass):
+    """Test setup with configuration missing required entries."""
+    with assert_setup_component(0):
+        assert await async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6309587)